src/main/resources/templates/in-app/placement-useful-information/v1.0.0.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <title>Deferral</title>
+</head>
+<body>
+  <h1>In-App Notification</h1>
+  <h2>Subject</h2>
+  <th:block th:fragment="subject">Deferral</th:block>
+  <h2>Content</h2>
+  <th:block th:fragment="content">
+    <p>
+      This notification pertains to your placement in
+      <th:block th:text="${not #strings.isEmpty(specialityName)} ? ${specialityName} : _"
+      >(Speciality name missing)
+      </th:block
+      >
+      at
+      <th:block th:text="${not #strings.isEmpty(siteKnownAs)} ? ${siteKnownAs} : _"
+      >(Placement name missing)
+      </th:block
+      >
+      starting on
+      <th:block th:text="${startDate} ? |${#temporals.format(startDate, 'dd MMMM yyyy')}| : _"
+      >(start date missing)
+      </th:block
+      >
+      .
+    </p>
+    <p>
+      Useful Information
+
+      <th:block: th:switch="${localOfficeContactType}">
+        <th:block th:case="NON_HREF"
+        >Further information on the management of specific issues can be found at your local offices
+          website, for example:
+        </th:block
+        >
+        <th:block th:case="*"
+        >Further information on the management of specific issues can be found <a
+          href="${localOfficeContactType} == 'email' ? |mailto:${localOfficeContact}"
+          th:target="_blank">
+          here</a>. for example:
+        </th:block>
+      </th:block:>
+
+    <ul>
+      <li>Taking time out of programme (OOP)</li>
+      <li>Working less than full time (LTFT)</li>
+      <li><a
+        href="https://www.hee.nhs.uk/our-work/supporting-doctors-returning-training-after-time-out">
+        Supported Return to Training (SuppoRTT)</a></li>
+      <li>Concerns or complaints</li>
+      <li>Specialty specific guidance</li>
+    </ul>
+
+    Please refer to the National guidelines found below:
+
+    <ul>
+      <li>‘Gold guide’ to Specialty Training found <a
+        href="https://www.copmed.org.uk/publications/gold-guide" th:target="_blank">
+        here</a>.
```

Comment:
I'd suggest tweaking this to be

<li><a
        href="https://www.copmed.org.uk/publications/gold-guide" th:target="_blank">
        Gold guide</a> to Specialty Training.
</li>